Cargo.toml

```diff
@@ -84,3 +95,6 @@ name = "slatedb"
 path = "src/cli/main.rs"
 required-features = ["cli"]
 
+[[bench]]
+name = "bloom_bench"
+harness = false
```

Comment:
this is false because we're using criterion?

Reply:
Yep, `harness = false`, disables the default test harness and uses criterion's harness instead